### PR TITLE
Improve grammar, phrasing and readability for contributors.md

### PR DESCRIPTION
**Pull request by timlinux**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,23 +39,23 @@ Our base of operations for group discussions on this project: [Jupiter Web Site
 
 * 🚀 The current focus and priority is on [Milestone 1.0](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/milestone/1)
 * 📝 You can submit an idea or a feature by opening an [issue](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/issues).
-  * _Please do a quick search prior - maybe a relevant issue already exists_ 😉
-* 💡 When you feel inspired about a particular issue or feature, please propose your implementation idea/strategy for approval via  comments, which indicates you are dedicated to solving this issue, thus avoiding duplication of effort.
+* _Please do a quick search prior top opening an issue - maybe a relevant issue already exists_ 😉
```
Comment (ChanceM) [inline CONTRIBUTING.md:42]: I think this should be to rather than top.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -32,23 +39,23 @@ Our base of operations for group discussions on this project: [Jupiter Web Site
...
 
 ### What is the Pull-Request (PR) etiquette?
 
-* 💻 Don't be shy of creating a Work-In-Progress (WIP) PR. This is a great way to geat some feedback and maybe some direction on your work.
-  * ✅ Create a markdown checklist to indicate what's done and what's still missing ([see this example](https://github.com/JupiterBroadcasting/jupiterbroadcasting.com/pull/112))
-* ⚙ If you're closing a specific issue I encourage you to use [Github's closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to automatically close the issue once it the PR is merged.
-* 🚮 Feel free to delete your feature branch after it has been merged via the PR. There’s a button handy on the GitHub interface, and worst case they can be reverted easily.
+* 💻 Don't be shy to create a Work-In-Progress (WIP) PR. This is a great way to get some feedback and maybe some direction for your work.
```
Comment (ChanceM) [inline CONTRIBUTING.md:55]: I think I would drop to and add a comma after shy.

File: CONTRIBUTING.md (hunk 1 above)
Comment (timlinux) [reply CONTRIBUTING.md]: Ok done, thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (timlinux) [reply CONTRIBUTING.md]: Done

**Review by CGBassPlayer — APPROVED** [review]

**Comment by elreydetoda** [comment]
Thanks for checking this out @CGBassPlayer and @ChanceM. :partying_face: :tada: 

@ChanceM did you have any other outstanding requests or do you also approve this PR?

**Review by ChanceM — APPROVED** [review]

**Comment by elreydetoda** [comment]
Cool, since we've got 2 approvals merging changes. :tada:

**Pull request merged**